test/unit/lib/tasks/reporting_test.rb

```diff
@@ -0,0 +1,26 @@
+require "test_helper"
+require "rake"
+
+class ReportingRake < ActiveSupport::TestCase
+  setup do
+    @document_1 = create(:published_edition, body: "Some text 1")
+    @document_2 = create(:draft_edition, body: "Some text 2")
+    @document_3 = create(:published_edition, body: "Some other text 1")
+  end
+
+  teardown do
+    Rake::Task["reporting:matching_docs"].reenable
+  end
+
+  test "it prints the content IDs of the matching documents from published editions" do
+    assert_output(/#{@document_1.document.content_id}/) { Rake.application.invoke_task "reporting:matching_docs[Some text]" }
```

Comment:
I wondered if these could not use regex like `assert_output(@document_1.document.content_id) { Rake.application.invoke_task "reporting:matching_docs[Some text]" }`, but I don't know if there is a way to do `contains`

Reply:
Yeah, I couldn't find an equivalent method that didn't require a regex.